backend/src/sessionRoutes.ts

```diff
@@ -92,20 +93,17 @@ router.post('/defence/deactivate', handleDefenceDeactivation);
 router.post('/defence/configure', handleConfigureDefence);
 router.post('/defence/resetConfig', handleResetDefenceConfigItem);
 
-// emails
-router.post('/email/clear', handleClearEmails);
-
 // chat
 router.post('/openai/chat', handleChatToGPT);
 router.post('/openai/addInfoToHistory', handleAddInfoToChatHistory);
-router.post('/openai/clear', handleClearChatHistory);
 
 // model configurations
 router.post('/openai/model', handleSetModel);
 router.post('/openai/model/configure', handleConfigureModel);
 
 // reset progress for all levels
 router.post('/reset', handleResetProgress);
+router.post('/resetlevel', handleResetLevel);
```

Comment:
I was thinking of something like this for the endpoints. Originally I considered a path param for reset level, but it didn't seem quite right, plus this is nice as it's consistent:

```
POST /restart?currentLevel=N

POST /reset?level=N
```

I also note that the /level endpoint seems a touch redundant, and could definitely use a path param, but that's a separate matter.

```
GET /level?level=1
vs
GET /level/1
```

Reply:
I dislike using restart and reset to distinguish between resetting the whole game and resetting a single level - think it should be spelled out which does which. I'm also keen to keep the language consistent with the frontend. 

happy with other alternatives that do this. e.g:

```
POST /resetall?currentLevel=N

POST /reset?level=N
```

```
POST /reset/all?currentlLevel=N

POST /reset?level=n
```

Reply:
Well, there's little difference between resetall and reset and restart, so it's going to be confusing whichever of these we would pick...

How about "resetgame" or similar?

Alternatively, we could use my initial thought about using a path param combined with yours for reset all:

```
POST /reset/all?currentLevel=N
POST /reset/{level}
```

Then it comes down to making it work in Express; likely we need them defined in the above order, so that express looks for "all" before trying a path param for level.

Reply:
yeah I like this. Done